ubuntu/ubuntu.go

```diff
@@ -80,6 +80,10 @@ func Update() error {
 		_, err = gc.CloneOrPull(url, dir, "master", false)
 		if err != nil {
```

Comment:
```
...
  if err == nil {
    break
  }
  log.Printf("failed to clone or pull: %s: %v", url, err)
  log.Printf("removing %s directory", cveTrackerDir)
  if err := os.RemoveAll(dir); err != nil {
    return xerrors.Errorf("failed to remove %s directory: %w", cveTrackerDir, err)
  }
}
```
looks cleaner to me here

Reply:
OK, let's change it to what you suggest.